site/_data/admin-rest-api-swagger.json

```diff
@@ -247,6 +247,26 @@
         }
       }
     },
+    "/broker-stats/topics" : {
```

Comment:
@lucperkins  can we hook the swagger update in the website build and delete this file (which is bound to always be outdated) from the git repo? 

I can update the Jenkins job config if needed.

Reply:
@merlimat I've updated the site generation script to include Swagger definition generation and removed the generated JSON from Git

Reply:
Nice!